.ci/xtask/src/core/metadata.rs

```diff
@@ -12,3 +13,10 @@ lazy_static! {
 pub fn cargo() -> cargo_metadata::Metadata {
     CARGO_METADATA.to_owned()
 }
+
+pub fn repository_url() -> String {
+    let carl_package: Package = cargo().workspace_packages().into_iter()
+        .find(|&package| package.name == "opendut-carl").cloned()
```

Comment:
As far as I know, you could get rid of that `cloned()` since you do a `to_owned` in the end, which does the same.

Reply:
True. Removed it.